06-building-a-program-start-to-finish/delivering-your-product/output/README.md

```diff
@@ -0,0 +1,5 @@
+#### Output
+- [Link to release note](https://docs.google.com/document/d/1sr2edp2YX0ghNikqTrBpTL8lwAvo4nONSY9D9GnsGus/edit)
+
+#### Author
+Owonikoko Oluwaseun S.
```

Comment:
I believe this should also be added to your repo's wiki

[Delivering your product](https://github.com/andela/test-simulations-mgmt-repo/tree/bayo/06-building-a-program-start-to-finish/delivering-your-product)